tests/fixtures.py

```diff
@@ -0,0 +1,145 @@
+#  Copyright (c) 2022 spdx contributors
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from datetime import datetime
+
+from src.model.actor import Actor, ActorType
+from src.model.annotation import Annotation, AnnotationType
+from src.model.checksum import Checksum, ChecksumAlgorithm
+from src.model.document import CreationInfo, Document
+from src.model.external_document_ref import ExternalDocumentRef
+from src.model.extracted_licensing_info import ExtractedLicensingInfo
+from src.model.file import File, FileType
+from src.model.license_expression import LicenseExpression
+from src.model.package import Package, PackageVerificationCode, PackagePurpose, ExternalPackageRef, \
+    ExternalPackageRefCategory
+from src.model.relationship import Relationship, RelationshipType
+from src.model.snippet import Snippet
+from src.model.version import Version
+
+"""Utility methods to create data model instances. All properties have valid defaults, so they don't need to be 
+specified unless relevant for the test."""
```

Comment:
I have implemented such a class for validation tests, too. After merging, we should see whether we can unify our approaches.